### PR TITLE
Update php-fpm-healthcheck

### PR DESCRIPTION
**Pull request by renatomefi**
# Usabilla PHP Docker Template

Reviewers: @usabilla/oss-docker

## Type

Please specify the type of changes being proposed:

| Q                 | A
| ----------------- | -----
| Dockerfile change?| yes

## Changelog

- Update php fpm healthcheck script to leverage listen queue metric, more at: https://github.com/renatomefi/php-fpm-healthcheck/pull/25

### CODE REVIEW

File: src/php/utils/docker/php-fpm-healthcheck [hunk 1]
```diff
@@ -50,7 +50,7 @@ export SCRIPT_FILENAME="/status"
 FCGI_CONNECT_DEFAULT="localhost:9000"
 
 # Required software
-command -v cgi-fcgi 1> /dev/null || { >&2 echo "Make sure fcgi is installed (i.e. apk add --no-cache fcgi). Aborting."; exit 4; }
+FCGI_CMD_PATH=$(command -v cgi-fcgi) || { >&2 echo "Make sure fcgi is installed (i.e. apk add --no-cache fcgi). Aborting."; exit 4; }
```
Comment (agustingomes) [inline src/php/utils/docker/php-fpm-healthcheck]: what would be the expected output for `FCGI_CMD_PATH`? I'd assume would be something like `/usr/bin/cgi-fcgi`

Comment (renatomefi) [reply src/php/utils/docker/php-fpm-healthcheck]: @agustingomes depends on the system, that's why we use command, but yes!

File: src/php/utils/docker/php-fpm-healthcheck [hunk 2]
```diff
@@ -77,7 +77,7 @@ get_fpm_status() {
 check_fpm_health_by() {
     OPTION=$(echo "$1" | sed 's/--//g; s/-/ /g;')
     VALUE_EXPECTED="$2";
-    VALUE_ACTUAL=$(echo "$FPM_STATUS" | grep "^$OPTION" | cut -d: -f2 | sed 's/ //g')
+    VALUE_ACTUAL=$(echo "$FPM_STATUS" | grep "^$OPTION:" | cut -d: -f2 | sed 's/ //g')
```
Comment (agustingomes) [inline src/php/utils/docker/php-fpm-healthcheck]: this change is also as consequence of what's described in [this PR](https://github.com/renatomefi/php-fpm-healthcheck/pull/25) right? so the option doesn't need to be given with the `:` suffixced to it

Comment (renatomefi) [reply src/php/utils/docker/php-fpm-healthcheck]: There's no consequence, basically this is what extracts the metric from php-fpm.
Before the regex would match `listen queue len:` and `listen queue:`, by adding the `:` to the regex it'll match the correct metric. The effect before is that wouldn't matter whether you'd be using one metric or the other, it'd always match the first one

Comment (agustingomes) [reply src/php/utils/docker/php-fpm-healthcheck]: aaaaaaaaah, because `listen queue` would also match `listen queue len` metrics. got it 👍

**Pull request merged**